services/121-service/src/programs/programs.service.ts

```diff
@@ -154,37 +155,12 @@ export class ProgramService {
     const programIds = user.programAssignments.map((p) => p.program.id);
     const programs = await this.programRepository.find({
```

Comment:
This is now a bit confusing. I would maybe rename the variables.

```suggestion
    const userProgramIds = user.programAssignments.map((p) => p.program.id);
    const programIds = await this.programRepository.find({
```